lib/srv/reexec.go

```diff
@@ -245,7 +242,7 @@ func RunCommand() (io.Writer, int, error) {
 
 	if uaccEnabled {
 		uaccErr := uacc.Close(c.UaccMetadata.UtmpPath, c.UaccMetadata.WtmpPath, tty)
-		if uaccErr != nil && !trace.IsAccessDenied(uaccErr) && !trace.IsNotFound(uaccErr) {
```

Comment:
So the change above makes uacc.Open best-effort, but this change here makes the condition more strict (i.e. we return any error now whereas previously access-denied/not-found were ignored). Is that intended?

Reply:
This was intentional on my part. In my mind, if Open passes there should be no good reason for Close to fail, and we should return the error to let the user know that their user accounting database may be in a bad state.